packages/otel/src/lib/observability-attributes.ts

```diff
@@ -0,0 +1,14 @@
+export const ObservabilityAttributes = {
+  SALEOR_API_URL: "saleorApiUrl",
+  SALEOR_VERSION: "saleorVersion",
+  CHANNEL_SLUG: "channelSlug",
+  TRANSACTION_ID: "transactionId",
+} as const;
+
+export enum AttributeNames {
```

Comment:
suggestion: maybe we should indicate these attributes are for GraphQL?

```suggestion
export enum GraphQLAttributeNames {
```